apps/wallet-dashboard/components/transactions/index.ts

```diff
@@ -1,15 +1,11 @@
 // Copyright (c) 2024 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-export { default as GasSummary } from './GasSummary';
 export { default as TransactionAmount } from './TransactionAmount';
 export { default as StakeTransactionCard } from './StakeTransactionCard';
 export { default as UnstakeTransactionCard } from './UnstakeTransactionCard';
 export { default as TransactionSummary } from './TransactionSummary';
-export { default as TransactionTile } from './TransactionTile';
+export * from './TransactionTile';
```

Comment:
wouldn't it be better to use the same export format for all of them?

Reply:
This is something we already agreed on doing, we shouldn't use `export { default ...`, but I won't change them in this PR since it's not breaking anything. I would have to change other 7 files and the PR is already long enough

https://github.com/iotaledger/iota/issues/4375

Reply:
perfect!